cmake/FindMetaExamples.cmake

```diff
@@ -35,8 +35,6 @@ function(get_excluded_meta_examples)
       statistical_testing/quadratic_time_maximum_mean_discrepancy.sg
       gaussian_process/classifier.sg
       gaussian_process/regression.sg
-      binary/svmlin.sg
```

Comment:
this is not the right fix here imo

This list is excluded meta examples (in this case for GPL)

Instead, you need to add the same names to the SVMLight exclusion section.

Reply:
I see what you mean. However, the problem here is caused by the fact that those two files don't exist anymore, so even if I moved them into the SVMLight section I would face the same issue (cmake refuses to run because it doesn't find those files).

Reply:
I see. I guess they were renamed?

Reply:
If so, we should just change the fname. IF they were deleted, then this can go..... but if they were just renamed the build should fail as the examples with the new name are not excluded from the tests and then produce a failure due to missing svmlight

Reply:
I did a bit of research around the repo and the only file that matches is an undocumented python exampled (examples/undocumented/python/graphical/svmlin.py). I also tried to search for those files inside git history, but I found nothing. If we look at the buildbot (http://buildbot.shogun-toolbox.org:8080/#/builders/4/builds/175) we see that three months ago everything was working. I tried to search for relevant changes in the last three months, but again I found nothing. I guess we can remove those files since they seem to have been not renamed.

Reply:
The buildbot will spot errors (hopefully) if we missed something anyway.